go-controller/pkg/cni/types/types.go

```diff
@@ -45,6 +45,15 @@ type NetConf struct {
 	LogFileMaxAge int `json:"logfile-maxage"`
 }
 
+// NetConfList describes an ordered list of NetConf
+type NetConfList struct {
```

Comment:
What are we using this type for?

Reply:
nice catch, not used anymore from what i see,
used in the previous implementation of this PR, will remove
thanks